kde-apps/minuet/minuet-9999.ebuild

```diff
@@ -27,6 +27,7 @@ COMMON_DEPEND="
 	$(add_qt_dep qtwidgets)
 	media-libs/alsa-lib
 	>=media-sound/drumstick-1.0.1
+	>=media-sound/fluidsynth-1.1.6
```

Comment:
Minimum version operator is not needed, there is only 1.1.6 in gentoo main tree and the package was not bumped since a year...

 Please check cmake files on stable branch if it is needed for 16.08.